### PR TITLE
Embed subtitles by default

### PR DESCRIPTION
**Pull request by howardbaek**
Explanation is inline.

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -117,7 +117,7 @@ ui <- fluidPage(
       ),
       div(
         prettySwitch("burn_subtitle", "Embed Subtitles",
-                     status = "success", fill = TRUE),
+                     value = TRUE, status = "success", fill = TRUE),
```
Comment (howardbaek) [inline app.R:120]: Setting `value = TRUE` tells the initial value of the switch to be `TRUE`.

**Comment by carriewright11** [comment]
Thank you! anything we should be concerned about with the check failure?

**Comment by howardbaek** [comment]
I'll check out these failures.

**Comment by cansavvy** [comment]
@howardbaek How do we get authorization to see the gitlab logs?

**Pull request closed without merging**